public/modules/custom/grants_oma_asiointi/src/Plugin/Block/OmaAsiointiBlock.php

```diff
@@ -156,19 +93,19 @@ public static function create(
     array $configuration,
     $plugin_id,
     $plugin_definition,
-  ) {
-    return new static(
+  ): static {
+    return new self(
```

Comment:
The return type hit (`static`) does not match with the actual return type (`self`)